openapi.yml

```diff
@@ -137,7 +137,7 @@ paths:
                         message: some text
           description: A generic error
         '401':
```

Comment:
Ignore below changes. It was done by stoplight studio and it just replaces the quotation marks